src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -239,12 +243,15 @@ def test_WINNF_FT_S_SIQ_3(self):
             for channel in channels_pal[0]))
 
     # Checks for cbsd 2 (index 1)
-    self.assertChannelsContainFrequencyRange(channels_low[1], freq_range_low)
-    self.assertChannelsContainFrequencyRange(channels_high[1], freq_range_high)
+    self.assertChannelsOverlapFrequencyRange(channels_low[1], freq_range_low, True, False)
+    self.assertChannelsOverlapFrequencyRange(channels_high[1], freq_range_high, False, False)
+    for channel in channels[1]:
+      self.assertLessEqual(channel['frequencyRange']['highFrequency'],
+                           freq_range_all['highFrequency'])
```

Comment:
Just use 3700000000 directly since `freq_range_all` is otherwise unused.

Reply:
changed